### PR TITLE
remove response summary column

### PR DESCRIPTION
**Pull request by leangseu-edx**
**TL;DR -** Response column shouldn't be showing on in the summary header

JIRA: [AU-598](https://openedx.atlassian.net/browse/AU-598)

**What changed?**

- Stop showing summary on `label_summary` is `undefined`
- Update unit test for this case

Screen Shot 2022-04-01 at 8 33 43 AM

**Developer Checklist**

- [ ] Reviewed the [release process](https://github.com/edx/edx-ora2/blob/master/.github/release_process.md)
- [ ] Translations and JS/SASS compiled
- [ ] Bumped version number in [setup.py](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/setup.py#L39) and [package.json](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/package.json#L3)

**Testing Instructions**

- Turn on ESG to see new column
- [ ] Summary response shouldn't show in the header

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @openedx/content-aurora

### CODE REVIEW

File: openassessment/xblock/static/js/src/lms/oa_course_items_listing.js [hunk 1]
```diff
@@ -248,6 +247,8 @@ export class CourseItemsListingView {
     $section.find('.open-response-assessment-summary').empty();
 
     $.each(this._columns, (index, v) => {
+      // if labe_summary is undefine, do not show it
```
Comment (jansenk) [inline openassessment/xblock/static/js/src/lms/oa_course_items_listing.js:250]: label

File: openassessment/xblock/static/js/spec/lms/oa_listing.js [hunk 2]
```diff
@@ -298,45 +298,56 @@ describe("OpenAssessment.CourseItemsListingView", function() {
...
+                view.renderGrids(ora2responses);
+                var items = $('.staff-esg-link');
+                var itemsCount = Object.keys(oraCourseItems).length;
+                expect(items.length / itemsCount).toEqual(2);
 
-            for (let i = 0; i < items.length; i++) {
-                let item = items[0];
-                expect(item.href).toContain(data.CONTEXT.ORA_GRADING_MICROFRONTEND_URL);
-            }
+                for (let i = 0; i < items.length; i++) {
```
Comment (jansenk) [inline openassessment/xblock/static/js/spec/lms/oa_listing.js:323]: this loop looks like it's not doing anything?

**Review by jansenk — COMMENTED** [review]
looks good, some small comments

File: openassessment/xblock/static/js/src/lms/oa_course_items_listing.js (hunk 1 above)
Comment (jansenk) [reply openassessment/xblock/static/js/src/lms/oa_course_items_listing.js:250]: and "undefined"

File: openassessment/xblock/static/js/spec/lms/oa_listing.js (hunk 2 above)
Comment (leangseu-edx) [reply openassessment/xblock/static/js/spec/lms/oa_listing.js:323]: Just to check if the url get sub into the column properly. I could just check one of them, however there isn't much different in my opinion.

**Review by jansenk — APPROVED** [review]
i approb

**Review by jansenk — CHANGES_REQUESTED** [review]
Sorry, I actually don't approve. The ticket also asks to make the button not "clickable" when ESG is enabled, and to change the column name.
